package.json

```
@@ -30,7 +30,7 @@
   "license": "Apache-2.0",
   "main": "index.js",
   "engines": {
-    "node": "<=16.x"
+    "node": ">=16.x"
```

I think we want this to be more explicit.
https://docs.npmjs.com/cli/v8/configuring-npm/package-json#engines
```suggestion
    "node": ">=16.0.0"
```